engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js

```diff
@@ -33,6 +33,13 @@ angular.module('Bastion.content-views').controller('ContentViewCompositeContentV
             return deferred.promise;
         };
 
+        $scope.newerVersionExists = function (contentViewVersion) {
+            var found = _.find(contentViewVersion['content_view']['content_view_version_ids'], function(num) {
```

Comment:
Good point.  It is actually using the database id, which implies the most recent version published.  In the example above, that would mean 2.2 is newer than 3.0.